README.md

```diff
@@ -32,13 +32,7 @@ git clone --filter=blob:none https://github.com/flutter/samples.git
 
 See the [contributor's guide](CONTRIBUTING.md)!
 
-## Questions or issues?
-
-If you have a general question about one of these samples or how to adapt its
-techniques for one of your own apps, try one of these resources:
-
-* [The FlutterDev Discord](https://discord.gg/rflutterdev)
-* [StackOverflow](https://stackoverflow.com/questions/tagged/flutter)
```

Comment:
Is StackOverflow Flutter no longer a thing? Our community still seems to be engaged there?

Reply:
Also, the Discord still seems inhabited by community developers and google engineers